csharp/src/Microsoft.ML.OnnxRuntime/targets/netstandard/props.xml

```diff
@@ -109,6 +116,38 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <Visible>false</Visible>
     </None>
+
+    <!-- arm64 -->
+    <None Include="$(MSBuildThisFileDirectory)..\..\runtimes\win-arm64\native\onnxruntime.dll"
+          Condition="'$(PlatformTarget)' == 'ARM64'">
+      <Link>onnxruntime.dll</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Visible>false</Visible>
+    </None>
+    <None Include="$(MSBuildThisFileDirectory)..\..\runtimes\win-arm64\native\onnxruntime_providers_shared.dll"
```

Comment:
AFAIK - We don't have shared providers for ARM/ARM64, but we seem to include the shared providers dll in the nuget, so keeping this line